docs/api_static/script.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+function addLink(element) {
+  const a = document.createElement("a");
```

Comment:
How likely is it that an api reference geared towards web developers will be viewed with ancient browsers though? I figured it was a pretty safe assumption. But even if I convert to ES5, which is quick and easy to do, some of the APIs used might not be supported either. What should be used in place of `querySelectorAll` and `DOMContentLoaded`? This is what caused jquery to be created...